client/src/components/SideBar/Channels/MeetingUserList/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { MeetingUserListWrapper, MeetingUserProfileWrapper, MeetingUserProfile } from './style';
+
+interface Props {
+  meetingUser: any;
+}
+
+function MeetingUserList({ meetingUser }: Props) {
+  return (
+    <>
+      {meetingUser &&
+        meetingUser[0].map((user: any) => {
+          return (
+            <MeetingUserListWrapper>
+              <MeetingUserProfileWrapper>
+                <MeetingUserProfile src="/images/default_profile.png" alt="profile" />
+              </MeetingUserProfileWrapper>
+              <p>{user.username}</p>
```

Comment:
```suggestion
              <p>{`${user.username}(${user.loginID})`}</p>
```
괄호치고 로그인아이디도 표시해주면 좋을 것 같아! 닉네임 중복이 가능하니까!